### PR TITLE
Payment system 'Stripe'.

### PR DESCRIPTION
**Pull request by ruslanshakirov**

### CODE REVIEW

File: app/assets/javascripts/reader_subscription.coffee [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$(document).ready ->
```
Comment (vladimirbazhanov) [inline app/assets/javascripts/reader_subscription.coffee]: coffee engine has shortcut for this

$ ->
  console.log 'document ready'

more here https://css-tricks.com/jquery-coffeescript/

File: app/controllers/subscription_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    customer = Stripe::Customer.retrieve(current_user.customer_id)
+    @subscription = customer.subscriptions.retrieve(current_user.subscription.plan)
+  end
+
+  def new
+    @stripe_list = Stripe::Plan.all
+    @plans = @stripe_list[:data].reverse
+
+    respond_to do |format|
+      format.js   {}
```
Comment (vladimirbazhanov) [inline app/controllers/subscription_controller.rb]: {} can be removed

File: app/controllers/webhooks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        user = User.find_by(customer_id: event_object)
+        user.role = :new_user
+        user.save
+      end
+      if event_json["type"] == "customer.subscription.created"
+        user = User.find_by(customer_id: event_object)
+        user.role = :reader
+        user.save
+      end
+    rescue StandardError => ex
```
Comment (vladimirbazhanov) [inline app/controllers/webhooks_controller.rb]: pls use new ruby 2.0 hash syntax

File: app/views/subscription/index.html.slim [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.row#my_subscription_page
+  .medium-12.columns.panel
+    h1 Subscription type: #{@subscription.plan.name}
```
Comment (vladimirbazhanov) [inline app/views/subscription/index.html.slim]: @subscription.plan.name should be in decorator

File: app/views/subscription/index.html.slim [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    h2 #{(@subscription.plan[:amount]/100.0)}$ for #{@subscription.plan.interval_count} #{@subscription.plan.interval}
```
Comment (vladimirbazhanov) [inline app/views/subscription/index.html.slim]: this should be in decorator too

and why you divide this by 100 here? float operations can provide problems

Comment (ruslanshakirov) [inline app/views/subscription/index.html.slim]: oh yes, thank you, this line is temporary
I'll change it how it should be

**Pull request merged**